editor/editor_inspector.cpp

```diff
@@ -911,6 +985,14 @@ void EditorProperty::_update_pin_flags() {
 	}
 }
 
+void EditorProperty::_update_configuration_warnings() {
+	if (Node *node = Object::cast_to<Node>(object)) {
```

Comment:
Currently we have a convention not to use this type of construct with an assignment in an `if` condition, so this should be:
```suggestion
	Node *node = Object::cast_to<Node>(object)
	if (node) {
```